docs/usage.md

```diff
@@ -6,58 +6,157 @@
+5. **Report**: Reports about the collected information of the runs are reported in a shiny app and a summary table in multiqc. Skip by passing `--skip_shiny` and `--skip_multiqc`.
+
+### 1. INPUT FILES SUMMARY
+
+Summary information about the input fasta files are calculated. Skip by `--skip_stats`.
+
+1. Sequence similarity. Calculates pairwise and average sequence similarity is calculated using TCOFFEE. Activate with `--calc_sim` (default: false).
+2. General summary. Calculates the number and the average length of sequences. Activate with `--calc_seq_stats` (default: true).
+3. Extract plddt. If the structures were generated by AF2, plddt is extracted and reported. Activate with `--extract_plddt` (default: false).
+
+### 2. GUIDE TREES
+
+Guide trees define the order in which sequences and profiles are aligned and hold a crucial role in determining the final MSA accuracy. Tree rendering techniques most commonly rely on pairwise distances between sequences.
+
+> **Note**
+> None of the below listed aligner needs an explicit definition of a guidetree: if they need one, they compute their own default guide tree. This explicit definition of a guide tree is available in case you want to test non-default combination of guide trees and aligner methods.
+
+Available GUIDE TREE methods (Optional):
+
+- [CLUSTALO](http://clustal.org/omega/#Documentation)
+- [FAMSA](https://github.com/refresh-bio/FAMSA)
+
+### 3. ALIGN
+
+Available ALIGN methods:
+
+**SEQUENCE-BASED** (only require a fasta file as input):
+
+- [CLUSTALO](http://clustal.org/omega/#Documentation) (accepts guide tree)
+- [FAMSA](https://github.com/refresh-bio/FAMSA) (accepts guide tree)
+- [KALIGN](https://github.com/TimoLassmann/kalign)
+- [LEARNMSA](https://github.com/Gaius-Augustus/learnMSA)
+- [MAFFT](https://mafft.cbrc.jp/alignment/server/index.html)
+- [MAGUS](https://github.com/vlasmirnov/MAGUS) (accepts guide tree)
+- [MUSCLE5](https://drive5.com/muscle5/manual/)
+- [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html) (accepts guide tree)
+
+**SEQUENCE- and STRUCTURE-BASED** (require both fasta and structures as input):
+
+- [3DCOFFEE](https://tcoffee.org/Projects/expresso/index.html) (accepts guide tree)
+
+**STRUCTURE-BASED** (only require stuctures as input):
+
+- [MTMALIGN](https://bio.tools/mtm-align)
+
+### 4. EVALUATE
+
+Optionally, the produced MSAs can be evaluated. Skip with `--skip_eval`.
+
+**SEQUENCE-BASED** (no extra input required):
+
+1. Calculate number of gaps and its average across sequences. Activate using `--calc_gaps` (default: true).
+
+**REFERENCE-BASED**:
+
+The reference MSAs (see samplesheet) are be used to evaluate the quality of the produced MSA.
+
+2. Sum Of Pairs. Calculates the SP score using the [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html) implementation. Activate using `--calc_sp` (default: true).
```

Comment:
A part form the suggestion below, might be also nice to include a pointer to the different metrics definition if available
```suggestion
2. Sum Of Pairs (SP). Calculates the SP score using the [TCOFFEE](https://tcoffee.readthedocs.io/en/latest/index.html) implementation. Activate using `--calc_sp` (default: true).
```